RULES.md

```diff
@@ -150,6 +152,18 @@ Value of field `route_id` should exist in GTFS data.
 
 Value of field `agency_id` should exist in GTFS data.
 
+<a name="E036"/>
+
+### E036 - Stop with invalid parent `location_type`
+
+Value of field `location_type` of parent found in field `parent_station` in invalid.
```

Comment:
I would be good to expand on why it's invalid here and explicitly list the cases we check.